vendor/data/codelists/BEIS/benefitting_countries.yml

```diff
@@ -10,1142 +10,1347 @@ data:
   - name: Algeria
     recipient_code: "130"
     code: DZ
+    graduated: no
```

Comment:
Would it be better to have `true` and `false` for these values?

Reply:
Oh wait, this is YAML being weird and YAML-y isn't it? 🙄

Reply:
Yeah, they translate to true and false automagically, I like the language of 'Graduated? No' for the code list.